### ISSUE TITLE
Is this accuracy the same as the original matlab version?

### ISSUE THREAD

**Issue by hx013** [varunagrawal/tiny-faces-pytorch#2]
Hello, thanks for your wonderful and beautiful implementation of the tiny face. I want to know the accuracy is the same as the original or not.

**Comment by varunagrawal** [comment]
I still need to implement the validation code to generate the results so we can check for accuracy against the provided toolkit.
I'll get around to it next week, so I'll keep this issue open till then!

**Comment by feixiangdekaka** [comment]
> I still need to implement the validation code to generate the results so we can check for accuracy against the provided toolkit.
> I'll get around to it next week, so I'll keep this issue open till then!

I  will  always waiting for  your  nice  job! Thank  you!

**Comment by varunagrawal** [comment]
Okay, I did a major refactor and should be able to train the model this weekend, as well as get the validation code up and running.
 Expect results this weekend!

**Comment by varunagrawal** [comment]
Okay I can officially confirm that as per the latest master, the results are pretty close to that of the paper. The paper reports a hard set mAP of 0.909 and I get 0.87. Similar results for medium and easy sets.
